scripts/socks/socks.sh

```diff
@@ -18,8 +18,8 @@ AUTOCONFIG_PORT=9500
 # from https://stackoverflow.com/a/246128/4907881
 PROXY_PAC_LOCATION="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
-socks_add_key_cmd="ssh-add -K ${VA_SOCKS_KEYFILE:-~/.ssh/*gov*}"
-socks_del_key_cmd="ssh-add -d ${VA_SOCKS_KEYFILE:-~/.ssh/*gov*}"
+socks_add_key_cmd="ssh-add -K ${VA_SOCKS_KEYFILE:-~/.ssh/id_rsa_vagov}"
+socks_del_key_cmd="ssh-add -d ${VA_SOCKS_KEYFILE:-~/.ssh/id_rsa_vagov}"
```

Comment:
Since I've [setup ssh to use the keychain](https://apple.stackexchange.com/a/250572), I actually comment these lines out.

I wonder if we could edit [ssh/config](https://github.com/department-of-veterans-affairs/devops/raw/master/ssh/config) so that we wouldn't need this line.

My current setup doesn't require this script at all as I'm using [Core Tunnel](http://coretunnel.app) to establish the socks connection for me and I'm specifying the proxy.pac hosted on GitHub in my system proxy settings.

Reply:
Hmm - if we could edit the ssh/config so we wouldn't need it that would be great; I haven't figured out the right incantations to do so however.

Core Tunnel looks like a cool solution as well. I'm a bit uncomfortable having proxy.pac hosted on github however; I know it's a bit of a stretch but I don't much like the idea of asking a file on the internet whether to proxy a given request. I know it's not that much of a difference however.

Reply:
proxy.pac file is used only during tunnel initialization to configure the proxy - it's not accessed on each request. It's an interesting idea to just grab it directly from github instead of running a local web server. There might be situations thought when folks might want to edit the rules in that file without pushing the changes where they would affect everyone (I've done it when needed access to more domains).

Reply:
I'm hosting mine in a gist so if you want to diverge from repo, you can just fork/host else where.

I think proxy.pac is accessed more often than once but less often than every request. Maybe when the system boots or the interface goes down/up. I'm not certain of that though.